packages/shylu/fastilu/src/shylu_fastilu.hpp

```diff
@@ -1187,14 +1182,17 @@ class MemoryPrimeFunctorN
         KOKKOS_INLINE_FUNCTION
             void operator()(const Ordinal index) const
             {
+                //bmk: fix unused warnings?
+                //does this functor actually have any side effects, or is it just incomplete?
```

Comment:
This is silly debug test to make sure the memory is in the device. May be all three of Prime* functors have no place in release mode, just in debug mode.